### ISSUE TITLE
accessibility of heading order

### ISSUE THREAD

**Issue by btopro** [McGarvelcuddy/monopoly-card#3]
https://github.com/McGarvelcuddy/monopoly-card/blob/26a78f3f4d51f5fdd3cac6f330c6976222d56a03/src/MonopolyCard.js#L68-L69

Look up heading order. H1-h6 is for accessibility purposes and have to be "in order". An h4 before an h1 is an a11y issue . Neither should be headings most likley and should instead be designed / css applied to div's with classes that help them look like headings while not being them semantically

**Comment by McGarvelcuddy** [comment]
Doing it as `<div class="foobar">Monopoly Card</div>` and `.foobar{ text-size: large;
}`
